### DISCUSSION TITLE
Proposal for HVML half-tag (For Chinese：关于HVML半标签的提案)

### DISCUSSION THREAD

**Discussion by situdesign** [HVML/PurC#36] [General]
&lt;hello&gt; This is called a TAG
&lt;hello | tail&gt; This is called a GROUP TAG
&lt;hello|, or |tail&gt; are called HALF TAG
Benefit: When modifying a large table, you can change only one cell, without refreshing the whole table.
Examples of usage:<br/>
&lt;name | age | gender | class | score&gt;
&lt;name|
&nbsp;  hvml code
&nbsp;   |name&gt;
&lt;age|
&nbsp;  hvml code
&nbsp;   |age&gt;
......
&lt;/name || score&gt; (double vertical lines indicate omission so that the closing tag does not have to be full spellings.)<br/>
Also, considering that the symbol '|' is tedious to type, you can use '\\' instead of '|'<br/>
&lt;name \ age \ gender \ class \ score&gt;
......
&lt;/name \\\\ score&gt;

For Chinese:
&lt;hello&gt; 这个叫标签
<hello | tail>这个叫组标签
<hello| 或 |tail>这个叫半标签
好处：修改一个大表格时，可以只改一格，不需要整体刷新。<br/>
用法举例：
"<name | age | gender | class | score>"
"<name|"
"  代码"
"   |name>"
"<age|"
"  代码"
"   |age>"
"……"
"</name || score> （双竖线表示省略，这样关闭标签不必全写）"<br/>
此外，考虑到符号“|”在输入时比较费事，可以用“\”代替“|”<br/>
"<name \ age \ gender \ class \ score>"
"……"
"&lt;/name \\\\ score&gt;"

**Comment by kenlsoft** [comment]
你这描述得不够清楚啊，不是很懂啊，什么叫可以只修改一格？

**Reply by VincentWei** [reply]
> 你这描述得不够清楚啊，不是很懂啊，什么叫可以只修改一格？

的确，我也没看懂。看不出来可以解决什么问题，用在什么场景下。

**Reply by situdesign** [reply]
举个例子，这样一个表格：
<table>
    <caption >标题</caption>
         <tr>
         	<th>2014年</th>
         	<th>2015年</th>
         	<th>2016年</th>
         </tr>
         <tr>
         	<td>GDP: 8000万</td>
         	<td>GDP: 10000万</td>
         	<td>GDP: 12000万</td>
         </tr>
</table>
如果我只修改GDP数值，是不是可以只请求这部分数据，不用去动年份？

如果用传统的做法，我需要给那个 标签增加一个 id 属性或 class 属性，然后先用选择器选中这个节点，然后 blablabla。使用了本提案所述的半标签之后，则可以写成：
"<gdp|"
" 代码"
" |gdp>"
